src/App.tsx

```diff
@@ -1,22 +1,27 @@
-import Index from './pages';
-import ActivitiesDetail from './pages/activities-detail';
-import { BrowserRouter, Routes, Route } from 'react-router-dom';
-import Layouts from './layouts';
-import './App.css';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Layouts from "./layouts";
+import routers, { DomTitle } from "./routers";
+
+import "./App.css";
 
 function App() {
-    return (
-        <div>
-            <BrowserRouter>
-                <Layouts>
-                    <Routes>
-                        <Route path="/" element={<Index></Index>}></Route>
-                        <Route path="/activities" element={<ActivitiesDetail></ActivitiesDetail>}></Route>
-                    </Routes>
-                </Layouts>
-            </BrowserRouter>
-        </div>
-    );
+	return (
+		<div>
+			<BrowserRouter>
+				<Layouts>
+					<Routes>
+						{routers.map((item, idx) => (
+							<Route
+								key={`routers-${idx}`}
```

Comment:
everytime when need a unique string can use `useId()` hook refer to [useid hook in react](https://javascript.plainenglish.io/the-useid-hook-in-react-18-take-a-look-19a6c2f645bc)